github_migration_process.md

```diff
@@ -0,0 +1,222 @@
+
+| **Area Leads**:   | Responsible for setting direction in specific areas of the project and triaging issues within their respective areas.|
+|-|-
+| **Committers**:   | Dedicated to working on the project, may have write access.|
+| **Contributors**: | Community members that make any contribution to the project, whether it’s submitting code, fixing or reporting bugs, writing docs, etc.|
+| **Users:**        | General users who have a need for the project.|
+
+> **As a measurement** we want to try and touch all new items within 48 hours. This is not exposed to the customer but is a metric for us to determine how quickly we are touching new issues. We need to ensure that feature area owners are actively triaging their issues and keeping them up to date.
+>
+> A defined group of individuals consisting of Area Leads or Developers will be responsible for reviewing all new items.
+>
+> Once a new item has arrived the defined resource for 'touching' these issues for the first time will set the label 'triage' to the issue so that it can be reviewed.
+>
+> Once the item is being reviewed, they will be assigned an 'Issue type' unless it requires additional investigation, then the 'investigation' label would be added to show there needs to be some additional work before routing the issue.
+
+Question Issues
+---------------
+
+> Once marked as a 'Question' the resource will assist with the issue by pointing to self-service options or provide customized help to resolve the issue. We like to have questions actively pushed out to SO and slack (once we build up the community).
+>  
+> ****Questions****
+>
+> ***'Optional' Phases will apply to issues that are immediately understood allowing them to bypass many of the phases, moving the bug right to 'in-progress'.***
+>
+> If the issue is not clear, the resource changes the label to 'needs-more-info' so the person who created the issue understands there needs to be additional engagement by them in order to get help.
+>
+> **Best Practice!**
+>
+> If the question can be handled in a quick exchange, then there is no need to add the 'in progress label. If there is some research or it will take some time to resolve the issue the use the 'in progress label.
+>
+> At this point the resource just needs to complete the request and close out the question once completed. There is not a label for this state since 'closed' is its own unique state.
+>
+> If it is determined the 'Question' is actually a 'Bug' or 'Enhancement' the 'Question' label will be removed and updated accordingly and assigned (if necessary) to the appropriate resource.
+
+Defects, Enhancement, Doc issues
+--------------------------------
+
+> After being 'triaged' and defined as either a ‘Code-Defect’, ‘Test-Defect’, or 'Enhancement' the item will need to be reviewed and have the following labels applied based on the review of the issue.  
```

Comment:
"the item will need to be reviewed and have the following labels applied based on the review of the issue".  This again reads like you're enforcing a specific idyllic flow between states.  It will get too process-laden and heavy to attempt to enforce a specific sequence of events.  Instead focusing on the fact that issues should be described accurately is simpler and more natural.